src/panels/lovelace/views/get-view.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  LovelaceViewConfig,
+  LovelaceViewElement,
+} from "../../../data/lovelace";
+import { createLovelaceElement } from "../create-element/create-element-base";
+import "./hui-masonry-view";
+
+const ALWAYS_LOADED_LAYOUTS = new Set(["masonry"]);
+
+const LAZY_LOAD_LAYOUTS = {
+  panel: () => import("./hui-panel-view"),
+};
+
+export const getLovelaceViewElement = (
```

Comment:
Move this to `lovelace/create-element` and rename this to align with the others `create-view-element`